msteams-platform/bots/build-a-bot.md

```diff
@@ -0,0 +1,58 @@
+| 3. | Bot registration service | - Azure AD <br> - Developer Portal |
+| 4. | Advanced bot capabilities | - Call and media bot <br> - Access data using MS Graph <br> - SSO <br> - Bot on Mobile |
+
+## Build a bot
+
+Let's start by building a conversational bot. You can use Teams Toolkit, Bot Framework SDK, or Teams AI library. After you build a conventional Teams bot app, you can add the AI layer to your bot.
+
+### Build a conventional Teams bot app
+
+| # | I want to build ... | Here's a sample |
+| --- | --- | --- |
+| 1. | Conversational bot using Bot Framework SDK | - [Bot app with Node.js](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/nodejs) <br> - [Bot app with C#](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/csharp) <br> - [Bot app with Python](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-conversation/python) |
+| 2. | Conversational bot using Teams Toolkit | [Create Teams conversation bot](../sbs-teams-conversation-bot.yml) |
+
+### Build an AI-powered Teams bot app
+
+You can build an AI bot or elevate your existing conventional bot to be powered by AI.
```

Comment:
I'd add the word "new" or "from scratch" for the first part, like, "You can build a new AI bot, or convert your existing conventional bot to be powered by AI".